NotifyMe/serializers.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from django.db import IntegrityError
+from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
+from rest_framework.exceptions import ValidationError
+from django.db import IntegrityError, DatabaseError
+from rest_framework import serializers
+from .models.notification import Notification 
+from .models.subscription import Subscription
+from .models.subscriptionPlan import SubscriptionPlan
+from .models.user import User
+from .models.notificationType import NotificationType
+from .constants import *
+from NotifyMe.services.service import UserService
+from NotifyMe.constants import NotificationTypeId 
+
+logger = logging.getLogger(__name__)
+
+class UserSerializer(serializers.ModelSerializer):
+    subscription_plan = serializers.PrimaryKeyRelatedField(queryset=SubscriptionPlan.objects.all())
+
+    class Meta:
+        model = User
+        fields = '__all__'
+        depth = 1
+
+    def create(self, validated_data):
+        user_service = UserService()
+        try:
+            return user_service.create_user(validated_data)
+        except IntegrityError as e:
+            error_message = str(e)
+            raise ValidationError(f"IntegrityError: {error_message}")
```

Comment:
just throw that exception object why are converting to ValidationError object change it everywhere